src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -616,6 +617,36 @@ args but inject to your defined parameter with another name.::
         }
     }
 
+Actions and file uploads
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.2
+
+    The ability to pass arguments to actions was added in version 2.2.
+
+If you want live component to track and send files you first need
+to mark file upload inputs as ``file`` Stimulus target.::
+
+.. code-block:: twig
+
+    <input type="file" {{ stimulus_target('live', 'file') }} name="my_file" />
+
+Then, when defining action you need to use special ``files(name)`` modifier.::
+
+.. code-block:: twig
+
+    <div
+        data-action="change->live#action"
+        data-action-name="prevent|files(my_file)|upload"
```

Comment:
```suggestion
        data-action-name="upload_files(my_file)|handleMyFileUpload"
```

So, I changed a few things:

A) No `prevent`. Not needed here, and I think it takes away focus.
B) I like `upload_files` as the modifier name. It feels like a "verb", which "feels" correct here. I want to "trigger the action of uploading"
C) Made the live action name super explicit that it is a custom thing for this example. At first, I was thinking `|upload` was another modifier 🙃